catalog/core/catalog-core-localstorageprovider/src/main/java/org/codice/ddf/catalog/content/impl/FileSystemStorageProvider.java

```diff
@@ -463,6 +461,14 @@ private ContentItem readContent(URI uri) throws StorageException {
       throw new StorageException("Cannot read " + reference + ".", e);
     } catch (EncryptorDecryptorException e) {
       throw new StorageException("Cannot decrypt the file content.", e);
+    } finally {
+      try {
+        fileInputStream.close();
```

Comment:
✏️ You can avoid the belt-and-suspenders of closing both the `fileInputStream` and decorator-enhanced `encrypytedInputStream` and just close the latter. The [`InputStream.close()` method](https://docs.oracle.com/javase/8/docs/api/java/io/InputStream.html#close--) specifies that it
> Closes this input stream and releases any system resources associated with the stream.

Those resources include those streams the implementation might decorate.